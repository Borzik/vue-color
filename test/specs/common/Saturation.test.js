import { shallow } from 'vue-test-utils'
import { createRenderer } from 'vue-server-renderer'

import Saturation from '../../../src/components/common/Saturation'

describe('Saturation.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(Saturation)
  })

  test('renders correctly', () => {
    const renderer = createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
