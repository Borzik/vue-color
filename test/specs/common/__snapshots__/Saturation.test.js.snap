// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Saturation.vue renders correctly 1`] = `
<div class="vc-saturation" style="background:hsl(150, 100%, 50%);">
    <div class="vc-saturation--white"></div>
    <div class="vc-saturation--black"></div>
    <div class="vc-saturation-pointer" style="top:70.80392156862746%;left:67.53246753246754%;">
        <div class="vc-saturation-circle"></div>
    </div>
</div>
`;
